WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizerTests.swift

```diff
@@ -19,9 +20,17 @@ class RemoteOrderSynchronizerTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
+
+        WordPressAuthenticator.initializeAuthenticator()
```

Comment:
❓ Why is this necessary?

Reply:
When we run `RemoteOrderSynchronizerTests` tests separately (during development) without initializing `WordPressAuthenticator` it crashes.